server/src/main/java/io/druid/guice/http/JettyHttpClientModule.java

```diff
@@ -119,6 +119,8 @@ public HttpClient get()
 
       httpClient.setIdleTimeout(config.getReadTimeout().getMillis());
       httpClient.setMaxConnectionsPerDestination(config.getNumConnections());
+      httpClient.setMaxRequestsQueuedPerDestination(config.getNumRequestsQueued());
+      httpClient.setConnectTimeout(500);
```

Comment:
Ok, then what do you think about adding a static variable? It would be better for readability.